### ISSUE TITLE
Heatmap Rendering broken

### ISSUE THREAD

**Issue by Cellebyte** [go-gitea/gitea#5328]
- Gitea version (or commit ref): 1.6.0+rc1-105-g4c1f1f964 (ref: 4c1f1f96465e809161f7d634a07eb60b4511db35)
- Git version: Alpine git
- Operating system: Alpine Linux
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [ ] MySQL
  - [x] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:

## Description
build a docker image from the master today which is integrated with our MsSQL Database.

The database schema is Latin1_General_CI_AI.

The return code is 500 "Internal Server Error".

Do we use the wrong schema?
```json
{
    "message":"arg timestamp as int: strconv.ParseInt: parsing \"2018-11-13T00:00:00Z\": invalid syntax",
    "url":"https://godoc.org/github.com/go-gitea/go-sdk/gitea"
}
```

## Screenshots

![screenshot from 2018-11-13 16-03-51](https://user-images.githubusercontent.com/16540824/48421929-ccdfe480-e75d-11e8-9c87-b51d3719a641.png)

**Comment by lunny** [comment]
It should also be fixed by #5401

**Comment by Cellebyte** [comment]
> It should also be fixed by #5401

Will their be a backport to 1.6 or will it be released in 1.7?

**Comment by lafriks** [comment]
Heatmap is not part of 1.6.0 release
